proxy/src/test/java/net/md_5/bungee/util/BoundedHashMapTest.java

```diff
@@ -0,0 +1,51 @@
+package net.md_5.bungee.util;
+
+import com.google.common.collect.ImmutableList;
+import org.junit.Test;
+
+public class BoundedHashMapTest
```

Comment:
A little nit here. Should be BoundedHashSetTest

Reply:
Woops, thanks for finding it.